sunpy/timeseries/sources/noaa.py

```diff
@@ -36,8 +36,7 @@ class NOAAIndicesTimeSeries(GenericTimeSeries):
     Examples
     --------
     >>> import sunpy.timeseries
-    >>> import sunpy.data.sample  # doctest: +REMOTE_DATA
-    >>> noaa = sunpy.timeseries.TimeSeries(sunpy.data.sample.NOAAINDICES_TIMESERIES, source='NOAAIndices')  # doctest: +REMOTE_DATA
+    >>> noaa = sunpy.timeseries.TimeSeries("ftp://ftp.swpc.noaa.gov/pub/weekly/RecentIndices.txt", source='NOAAIndices')  # doctest: +REMOTE_DATA
```

Comment:
@hayesla Is this weird? I didn't think it made sense to include a call to Fido in the example section of a doc string.

Reply:
I actually think it's a nice example of how TimeSeries can also accept a URL!:)

Reply:
I actually think its a nice example of how TimeSeries can also accept a URL! :)